lib/lib.scriptHost.d.ts

```diff
@@ -1,294 +0,0 @@
-/*! *****************************************************************************

```

Comment:
OK, looks like the new spelling (all lowercase) is used everywhere, so this would be a FIX for a problem might have existed before!

[Jakefile.js  line 196   (looks same in aa53b9ee7b and master)](https://github.com/Microsoft/TypeScript/blob/aa53b9ee7b13bfedc59296b1ee4c19ffa9ae6bcd/Jakefile.js#L196)
```typescript
var hostsLibrarySources = ["dom.generated.d.ts", "webworker.importscripts.d.ts", "scripthost.d.ts"];

var librarySourceMap = [
        // Host library
        { target: "lib.dom.d.ts", sources: ["header.d.ts", "dom.generated.d.ts"] },
        { target: "lib.dom.iterable.d.ts", sources: ["header.d.ts", "dom.iterable.d.ts"] },
        { target: "lib.webworker.d.ts", sources: ["header.d.ts", "webworker.generated.d.ts"] },
        { target: "lib.scripthost.d.ts", sources: ["header.d.ts", "scripthost.d.ts"] },
```